src/storage/__init__.py

```diff
@@ -0,0 +1 @@
+from .storage import InventoryItem, InventoryStorage
```

Comment:
You are adding `__init__` here and remove it from `network`, using dotted imports here, but rely on pathmagic in other places. So inconsistent.

Reply:
I've (luckily? :)) never had to deal with this part of python so don't know what would be best/correct/portable fix. I'll try .stuff